src/felles-komponenter/skjema/input/input.js

```diff
@@ -29,19 +30,28 @@ InnerInputComponent.defaultProps = {
   input: undefined,
 };
 
-
-function Input({ feltNavn, ...rest }) {
+function Input({ feltNavn, bredde, datoFelt, ...rest }) {
+  const normaliserDatoFunksjon = datoFelt ? normaliserInputDato : null;
```

Comment:
Bruker Input for dato-input også. Dermed trengs en prop for å se om input-verdi skal vasker og normaliseres som dato.